src/layers/domain/core/cpm_system_id/v1.py

```diff
@@ -84,3 +84,19 @@ def validate_key(cls, key: str) -> bool:
             return False
         ods_code, number = parts
         return ods_code.isalpha() and number.isdigit() and len(number) == 6
+
+
```

Comment:
This can be ignored. It's just a mock class which will be overwritten once PI-495 is rebased